app/lib/actions/katello/repository/export.rb

```diff
@@ -0,0 +1,40 @@
+module Actions
+  module Katello
+    module Repository
+      class Export < Actions::EntryAction
+        middleware.use Actions::Middleware::KeepCurrentUser
+
+        input_format do
+          param :id, Integer
+          param :export_result, Hash
+        end
+
+        # @param repo
+        # @param since ISO8601 timestamp for computing incremental exports
+        # @param export_suffix suffix to use on export dir
+        def plan(repo, since = nil, export_suffix = 'repo_export')
```

Comment:
Correct, i'm using it in a later PR but for this specific PR it's always the setting set in the Admin/settings page and then `/repo_export` for the export location. I was not sure if it's better to add it now to keep the PR sizes down, or put in one bigger PR later that alters previous PRs :koala: